manifests/init.pp

```diff
@@ -90,6 +90,7 @@
 
   $enable_ostree = $katello::params::enable_ostree,
   $candlepin_event_queue = $katello_devel::params::candlepin_event_queue,
+  $candlepin_qpid_exchange = $katello_devel::params::candlepin_qpid_exchange,
```

Comment:
Strange, the candlepin_event_queue isn't in params since it inherits from katello